panther_analysis_tool/main.py

```diff
@@ -156,6 +156,9 @@ def load_analysis_specs(directories: List[str]) -> Iterator[Tuple[str, str, Any,
     loaded_specs: List[Any] = []
     for directory in directories:
         for relative_path, _, file_list in os.walk(directory):
+            # Skip hidden folders
+            if relative_path.split('/')[-1].startswith("."):
+                continue
```

Comment:
Weird edge case, this totally skips testing files in the current working directory (since the path is just `.` which happens to start with `.`). Maybe a slight modification here?
```suggestion
            # Skip hidden folders
            if relative_path.split('/')[-1].startswith(".") and relative_path != '.':
                continue
```

maybe that's a bit hacky, if you have a better suggestion all for it. But I'd like to preserve the functionality that running `panther_analysis_tool test` in a directory with detections tests all those detections.

Reply:
That makes sense since the first iteration of `os.walk` will yield `./` so it ignores `.`

The suggestion won't work since `relative_path` will be `./` (since we're not applying the same python splitting to this condition

```suggestion
            # Skip hidden folders
            if relative_path.split('/')[-1].startswith(".") and relative_path != "./":
                continue
```

Reply:
Whichever way works is fine with me, I was just testing a few cases locally and noticed it and tried this out to fix it and it seemed work. I'm sure there are some weird edge cases, not sure what the most definitive rule set would be to work perfectly in all cases.

Reply:
This should work now. Attached is a SS to show that `./` (the edge case which happens to actually be the first guaranteed response from `os.walk` is the first iteration)

![image](https://user-images.githubusercontent.com/71197790/116767192-a335d900-a9fc-11eb-8e2b-b6e41f13a1d8.png)

If there's a weird issue where this is not guaranteed on all *nix systems, we can always write the conditional as:

```
if relative_path.split('/')[-1].startswith('.') and len(relative_path.split('/')) != 1:
```

To basically assert that there's always something after the initial `./` but I think we should be safe -- the current implementation is small optimization that avoids an extra (likely unnecessary) `len` and `split` call each time